src/acl_kernel_if.cpp

```diff
@@ -44,7 +44,8 @@ acl_process_autorun_profiler_scan_chain(unsigned int physical_device_id,
 // to verify that the hardware and HAL have an identical view of the CSR
 // address map.
 #define CSR_VERSION_ID_181 (3)
-#define CSR_VERSION_ID (4)
+#define CSR_VERSION_ID_20222 (4)
```

Comment:
Since the versioning scheme changed from FPGA SDK to oneAPI anyway, how about using `_` inplace of `.`?
```suggestion
#define CSR_VERSION_ID_2022_2 (4)
```